x-pack/auditbeat/module/system/sockets/_meta/data.json

```diff
@@ -0,0 +1,40 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "metricset": {
+        "module": "system",
+        "name": "sockets",
+        "rtt": 115
+    },
+    "system": {
+        "sockets": {
+            "socket": [
```

Comment:
Same as for the users, if possible I would advice to not use nested documents.

Reply:
Without nested docs we can more easily map this to ECS, which I think we should do here. A good number of these fields can be changed to take advantage of ECS (I think only state and status do not have a place in ECS).

Reply:
Changed to ECS fields: https://github.com/elastic/beats/blob/1ed02be49277f3343cdb4c062092730a26013f23/x-pack/auditbeat/module/system/socket/_meta/data.json